sdk/operationalinsights/Azure.ResourceManager.OperationalInsights/api/Azure.ResourceManager.OperationalInsights.netstandard2.0.cs

```diff
@@ -228,10 +18,10 @@ public partial class LogAnalyticsQueryPackCollection : Azure.ResourceManager.Arm
     public partial class LogAnalyticsQueryPackData : Azure.ResourceManager.Models.TrackedResourceData
     {
         public LogAnalyticsQueryPackData(Azure.Core.AzureLocation location) : base (default(Azure.Core.AzureLocation)) { }
+        public System.DateTimeOffset? CreatedOn { get { throw null; } }
+        public System.DateTimeOffset? ModifiedOn { get { throw null; } }
         public string ProvisioningState { get { throw null; } }
-        public string QueryPackId { get { throw null; } }
-        public System.DateTimeOffset? TimeCreated { get { throw null; } }
-        public System.DateTimeOffset? TimeModified { get { throw null; } }
+        public System.Guid? QueryPackId { get { throw null; } }
     }
     public partial class LogAnalyticsQueryPackQueryCollection : Azure.ResourceManager.ArmCollection, System.Collections.Generic.IAsyncEnumerable<Azure.ResourceManager.OperationalInsights.LogAnalyticsQueryPackQueryResource>, System.Collections.Generic.IEnumerable<Azure.ResourceManager.OperationalInsights.LogAnalyticsQueryPackQueryResource>, System.Collections.IEnumerable
```

Comment:
Nit: Do we consider renaming this resource? QueryPackQuery is a little weird. Is it acceptable that we change it to `LogAnalyticsQuery`?

Reply:
I think it's acceptable as I don't see other types of queries.